vnext/Microsoft.ReactNative.Cxx/JSValueWriter.h

```diff
@@ -190,6 +190,16 @@ inline void WriteValue(IJSValueWriter const &writer, JSValueArray const &value)
   JSValue::WriteArrayTo(writer, value);
 }
 
+inline void WriteCustomDirectEventTypeConstant(
+    IJSValueWriter const &writer,
```

Comment:
Can you instead create helpers that just take an event name and automatically append `on` / `top`:
* `inline void WriteCustomDirectEventTypeConstant(IJSValueWriter const &writer, std::string_view eventName)`
* `inline void WriteCustomDirectEventTypeConstant(IJSValueWriter const &writer, std::wstring_view eventName)`

If you want then those helpers can call the more generic:
* `inline void WriteCustomDirectEventTypeConstant(IJSValueWriter const &writer, std::string_view propertyName, std::string_view registrationName)`
* `inline void WriteCustomDirectEventTypeConstant(IJSValueWriter const &writer, std::wstring_view propertyName, std::wstring_view registrationName)`